lib/feedjira/parser/json_feed.rb

```diff
@@ -8,7 +8,7 @@ class JSONFeed
       include FeedUtilities
 
       def self.able_to_parse?(json)
-        %r{https://jsonfeed.org/version/} =~ json
+        %r{https:\\?/\\?/jsonfeed.org\\?/version\\?/} =~ json
```

Comment:
I wonder if rather than making the regex more complicated, we could instead do basic string matching, since this doesn't appear to need a regex at all. E.g.:

```suggestion
        json.include?("https://jsonfeed.org/version/") ||
          json.include?("https:\/\/jsonfeed.org\/version\/")
```

another possibility would be to make it even more basic, like:

```suggestion
        json.include?("jsonfeed.org")
```

This whole approach seems a bit busted to me, since it might actually be an xml feed with the url in the content somewhere. What I really want to do is:

```suggestion
        JSON.parse(json)
        true
      rescue JSON::ParserError
        false
      end
```

However, that won't work when we're only passing in the first 4000 characters of the feed...

Reply:
That all makes sense to me.

Of the first two suggestions, I'd personally prefer the first one, as this seems to be safer - because it seems to me that the second suggestion is slightly more likely to result in false positives, e.g. when someone mentions "jsonfeed.org" in a blog post or similar.

Either way, I'm all for replacing the regex with string matching. Just more readable and obvious.

Reply:
Yeah, I'm fine with the first one. I agree, it gives us a bit more safety.

Reply:
So, I merged the suggestion, but now there are errors. The strings should be single-quote-strings I guess to avoid escaping swallow the escaping backslashes. I'll fix that (and might as well add a test, too).